views/emails/html/update-task.php

```diff
@@ -10,93 +10,92 @@
                                         $assign_users[] = esc_html( $assign->display_name );
                                     }
 
-                                    echo implode( ', ', $assign_users );
+                                    echo esc_html( implode( ', ', $assign_users ) );
                                 ?>
                             </p>
                         </td>
                         <td style="padding-left: 80px;">
-                            <p style="font-size: 16px; line-height: 30px; margin: 0; color: #7E8690;"><?php _e( 'Due Date', 'wedevs-project-manager' ); ?></p>
+                            <p style="font-size: 16px; line-height: 30px; margin: 0; color: #7E8690;"><?php esc_html_e( 'Due Date', 'wedevs-project-manager' ); ?></p>
                             <p style="font-size: 16px; color: #000000; line-height: 30px; margin: 0;">
                                 <?php
 
-                                    echo empty( $due_date['date'] ) ? '&#x02013;&#x02013;' : pm_date_format( esc_html( $due_date['date'] ) );
+                                    echo empty( $due_date['date'] ) ? '&#x02013;&#x02013;' : esc_html( pm_date_format( $due_date['date'] ) );
                                 ?>
                             </p>
                         </td>
                     </tr>
                     <tr>
                         <td colspan="2">
                             &nbsp;
-                            <p style="font-size: 16px; line-height: 30px; margin: 0; color: #7E8690;"><?php _e( 'Project', 'wedevs-project-manager' ); ?></p>
+                            <p style="font-size: 16px; line-height: 30px; margin: 0; color: #7E8690;"><?php esc_html_e( 'Project', 'wedevs-project-manager' ); ?></p>
                             <h2 style="font-size: 18px; color: #000; margin: 0;"><?php echo esc_html( $project_title ); ?></h2>
                         </td>
                     </tr>
 
                     <tr>
                         <td colspan="2">
                             &nbsp;
-                            <p style="font-size: 16px; line-height: 30px; margin: 0; color: #7E8690;"><?php _e( 'Description', 'wedevs-project-manager' ); ?></p>
+                            <p style="font-size: 16px; line-height: 30px; margin: 0; color: #7E8690;"><?php esc_html_e( 'Description', 'wedevs-project-manager' ); ?></p>
                             <div style="padding: 5px 15px; background: #edeef45e; border: 1px solid #f2f2f2; border-radius: 5px; margin-bottom: 30px; line-height: 26px; margin-top: 10px;">
-                                <?php echo empty( $description['html'] ) ? __( 'No description found!', 'wedevs-project-manager' ) : $description['html']; ?>
+                                Test Update task: <?php echo empty( $description['html'] ) ? esc_html_e( 'No description found!', 'wedevs-project-manager' ) : wp_kses_data( $description['html'] ); ?>
```

Comment:
Is this for test purpose? Found a similar thing above. If not, change the line as below:
```
<?php echo esc_html__( 'Test Update task: ', 'wedevs-project-manager' ) . empty( $description['html'] ) ? esc_html__( 'No description found!', 'wedevs-project-manager' ) : wp_kses_data( $description['html'] ); ?>
```